package.json

```diff
@@ -1,21 +1,18 @@
 {
-  "name": "react-native-voice",
+  "name": "@react-native-community/voice",
   "description": "React Native Native Voice library for iOS and Android",
   "version": "0.3.0",
   "author": "Sam Wenke <samwenke@gmail.com>",
-  "homepage": "https://github.com/react-native-community/react-native-voice",
+  "homepage": "https://github.com/react-native-community/voice",
   "devDependencies": {
-    "eslint": "^5.3.0",
-    "eslint-plugin-flowtype": "^2.50.0",
-    "eslint-plugin-import": "^2.13.0",
-    "eslint-plugin-react": "^7.10.0",
-    "eslint-plugin-react-native": "^3.2.1",
-    "pre-commit": "^1.2.2",
-    "prettier": "^1.12.0",
+    "@react-native-community/eslint-config": "^0.0.7",
+    "@react-native-community/eslint-plugin": "^1.0.0",
```

Comment:
I believe `@react-native-community/eslint-plugin` can be removed if we have the community eslint-config `0.0.7`, since the `eslint-plugin` is included as a dependency from `0.0.7`

Reply:
Great! Made the change